benchmarks/timeseries/benchmark_ringbuffer.py

```diff
@@ -117,25 +116,26 @@ def main() -> None:
 
     Result of previous run:
 
-    Date: Do 22. Dez 15:03:05 CET 2022
+    Date: Fr 27. Jan 16:08:50 CET 2023
     Result:
 
-           =========================================
-    Array: ........................................
-    List:  ........................................
+           ===========
+    Array: ..........
+    List:  ..........
     Time to fill 29 days with data:
-    Array: 0.09411649959984061 seconds
-    List:  0.0906366748000437 seconds
-    Diff:  0.0034798247997969156
-           =========================================
-    Array: ........................................
-    List:  ........................................
+            Array: 7.436426098603988 seconds
+            List:  7.500101384799928 seconds
+            Diff:  -0.06367528619593976
+           ===========
+    Array: ..........
+    List:  ..........
     Filling 29 days and running average & mean on every day:
-    Array: 0.09842290654996759 seconds
-    List:  0.1316629376997298 seconds
-    Diff:  -0.03324003114976222
+            Array: 7.6797407130012285 seconds
+            List:  8.115826181398006 seconds
+            Diff:  -0.43608546839677764
+
```

Comment:
Why do you think this is?  I can't believe creating a new `Sample` object for every message is making the benchmark 70-80x longer.

Maybe we should also try benchmarking the filling times and average/mean times separately?

Reply:
Yah, the previous tests were faster due to missing window not down completely or properly I think, so I don't think we can compare that